docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
+| [Apache Mahout](https://mahout.apache.org/)                            | - Data cleaning and processing  <br/> - Model building                                            | - Distributed linear algebra framework and mathematically expressive [Scala](https://www.scala-lang.org/api/current/) DSL <br/> - Support for multiple distributed backends (including [Apache Spark](https://spark.apache.org/))  <br/> - Modular native solvers for CPU/GPU/CUDA acceleration                                                                                                             |
+| [Apache Commons Math](https://commons.apache.org/proper/commons-math/) | - Data cleaning and processing  <br/> - Model building                                            | - Mathematics and statistics operations for Java <br/> - Correlations, distributions, linear algebra, geometry, and other operations <br/> - Machine learning models                                                                                                                                                                                                                                        |
+| [NM Dev](https://nm.dev/)                                              | - Data cleaning and processing  <br/> - Model building                                            | - Java math library of numerical algorithms <br/> - Object-oriented numerical methods <br/> -  Linear algebra, optimization, statistics, calculus, and more operations                                                                                                                                                                                                                                      |
+| [Apache OpenNLP](https://opennlp.apache.org/)                          | - Data cleaning and processing  <br/> - Model building                                            | - Machine-learning-based toolkit for the processing of natural language text <br/> - Tokenization, sentence segmentation, part-of-speech tagging, and other tasks <br/> - Built-in tools for data modeling and model validation                                                                                                                                                                             |
+| [Charts](https://github.com/HanSolo/charts)                            | Data exploration and visualization                                                                | - Scientific [JavaFX](https://openjfx.io/) charting library under development                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Note: The last update for this library was 2 years ago

Reply:
Let's keep